yt/frontends/adaptahop/definitions.py

```diff
@@ -0,0 +1,41 @@
+"""
+Data structures for AdaptaHOP
+
+
+
+"""
+
+#-----------------------------------------------------------------------------
+# Copyright (c) 2013, yt Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
```

Comment:
I think we're removing the copyright comment blocks now, right?

Reply:
nevermind I realized that your PRs removing them were for yt-4.0.

Reply:
That's actually a fair comment since #2396 has already been accepted. Either (1) I remove the copyright notices here, (2) I reopen a PR on yt-4 once #1838 has been merged in, (3) I rebase this PR on yt-4 without the copyright notices.

What do you think?

Reply:
I think (2) seems like the best option. There are other frontends that have been added to master that aren't on the 4.0 branch, so they'll need to be cleaned up too. I think it makes sense to do a cleanup of all of it once yt-project/yt#1838 is in.

Reply:
Fair enough! For reference, I made a mistake, the merge PR is #2172 (so that it appears in its log).

Reply:
Ha, clearly I haven't had my ☕️ yet!